pkg/asset/tls.go

```diff
@@ -31,13 +36,13 @@ func newTLSAssets(altNames tlsutil.AltNames) ([]Asset, error) {
 		return assets, err
 	}
 
-	kubeletKey, kubeletCert, err := newKubeletKeyAndCert(caCert, caKey)
+	kubeletKey, kubeletCert, err := newKubeletKeyAndCert(caCert, caPrivKey)
```

Comment:
The private key ends up being used to create new api and kubelet private keys and certificates signed by the CA ala. `tlsutil.NewSignedCertificate` and x509.CreateCertificate https://golang.org/pkg/crypto/x509/#CreateCertificate. This has the stipulation that the provided private key not have a passphrase right. I think that is fine, I just want to be sure I understand this.